src/IssueViz.UnitTests/IssueVisualizationControl/IssueVisualizationViewModelTests.cs

```diff
@@ -574,21 +574,16 @@ public void SelectionService_OnLocationChanged_NewLocationIsNull_NoNavigation()
         }
 
         [TestMethod]
-        [DataRow(true)]
-        [DataRow(false)]
-        public void SelectionService_OnLocationChanged_NewLocationIsNotNull_NavigabilityUpdated(bool navigationSucceeded)
```

Comment:
No longer relevant, there is no property to set anymore, and the span is set in LocationNavigator